src/components/Blocking/SpecificIssue.js

```diff
@@ -61,13 +62,21 @@ const styles = StyleSheet.create({
     marginLeft: 18,
     marginBottom: 19
   },
+  containerBtnText: {
+    marginRight: 22,
+    marginLeft: 18,
+    marginBottom: 19,
+    fontFamily: fonts.inter[500],
+    fontSize: 16,
+    color: COLORS.white2
+  },
   btn: {
     paddingLeft: 18,
     paddingRight: 22,
     paddingTop: 8
   },
   btnSkip: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in background color from `COLORS.lightgrey` to `COLORS.gray100` should be fine as long as it aligns with the overall design theme and doesn't affect readability or accessibility.



<!-- This is an auto-generated comment by OSS CodeRabbit -->